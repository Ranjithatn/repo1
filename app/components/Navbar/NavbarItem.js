import React from "react";

const NavbarItem = ({ className, id, children }) => {
  return (
    <div
      className={`${"navbar-item" + (className ? `${" " + className}` : "")}`}
      id={id}
    >
      {children}
    </div>
  );
};

export default NavbarItem;
