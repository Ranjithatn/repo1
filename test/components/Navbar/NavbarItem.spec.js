import React from "react";
import toJson from "enzyme-to-json";
import NavbarItem from "../../../app/components/Navbar/NavbarItem";
import Adapter from "enzyme-adapter-react-16";
import { configure, shallow, mount } from "enzyme";


configure({ adapter: new Adapter() });


describe('<NavbarItem />', () => {

  it("should render without crashing", () => {
    const component = mount(
      <NavbarItem />
    );
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });
  
});
