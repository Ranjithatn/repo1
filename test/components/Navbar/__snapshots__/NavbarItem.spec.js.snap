// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavbarItem /> should render without crashing 1`] = `
<NavbarItem>
  <div
    className="navbar-item"
  />
</NavbarItem>
`;
